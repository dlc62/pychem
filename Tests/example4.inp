[example4]
Method = "HF"
Basis_Sets = ["6-311G"]
Multiplicity = 1
Charge = 0
Coords = [['H', 1.0,  0.  ,  0.,   0.],
          ['H', 1.0,  1.0 ,  0.,   0.]]
Reference = "RHF"
Max_SCF_Iterations = 50

