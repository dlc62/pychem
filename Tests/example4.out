-------------------------------------------------------
 Hartree-Fock iterations 
-------------------------------------------------------

 Cycle:  1
 Total energy:  -1.0615920859
 Change in energy:  0.647182618353
 DIIS error:  0.128688632613

 Cycle:  2
 Total energy:  -1.09531326344
 Change in energy:  -0.0337211775467
 DIIS error:  0.0253495851699

 Cycle:  3
 Total energy:  -1.09668793292
 Change in energy:  -0.0013746694738
 DIIS error:  0.00217993487595

 Cycle:  4
 Total energy:  -1.0968651055
 Change in energy:  -0.000177172588731
 DIIS error:  7.55534759884e-05

 Cycle:  5
 Total energy:  -1.09686463292
 Change in energy:  4.7258021052e-07
 DIIS error:  4.67849085878e-07

-------------------------------------------------------
 End of Hartree-Fock iterations 
-------------------------------------------------------

Final HF energy:  -1.09686463292


 Alpha Orbital Energies 
[-0.52792029  0.12351641  0.32767441  0.55229706  2.53716176  2.65148538]

 Alpha MOs 
[[ 0.16103948  0.10410185  0.04534354 -0.09785364 -1.08632599  1.13312461]
 [ 0.30670019  0.07410511  0.82277378 -1.34614572  1.6429205  -1.21628134]
 [ 0.16692841  1.54967297 -0.88857946  1.86878758 -1.0472995   0.46949988]
 [ 0.16103948 -0.10410185  0.04534354  0.09785364  1.08632599  1.13312461]
 [ 0.30670019 -0.07410511  0.82277378  1.34614572 -1.6429205  -1.21628134]
 [ 0.16692841 -1.54967297 -0.88857946 -1.86878758  1.0472995   0.46949988]]

 Alpha Density Matrix 
[[ 0.02593372  0.04939084  0.02688206  0.02593372  0.04939084  0.02688206]
 [ 0.04939084  0.09406501  0.05119697  0.04939084  0.09406501  0.05119697]
 [ 0.02688206  0.05119697  0.02786509  0.02688206  0.05119697  0.02786509]
 [ 0.02593372  0.04939084  0.02688206  0.02593372  0.04939084  0.02688206]
 [ 0.04939084  0.09406501  0.05119697  0.04939084  0.09406501  0.05119697]
 [ 0.02688206  0.05119697  0.02786509  0.02688206  0.05119697  0.02786509]]


-------------------------------------------------------
 End of Hartree-Fock calculation 
-------------------------------------------------------

